larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -146,6 +167,18 @@ void* Init(TfLiteContext* context, const char* buffer, size_t length) {
     return conv_params;
   }
 
+  if (conv_params->padding_type == TfLitePadding::kTfLitePaddingSame &&
+      conv_params->pad_value == 0 &&
+      conv_params->activation != kTfLiteActNone) {
+    context->ReportError(context,
+                         "Fused activations are not supported in combination "
+                         "with zero-padding.");
```

Comment:
```suggestion
                         "Fused activations are only supported with valid or"
                         "one-padding.");
```

Reply:
:+1:  Reminder to self: this should also be changed in the unit test "death test" that expects this error message.